code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -667,6 +667,10 @@
 
 /datum/species/golem/clockwork/no_scrap //These golems are created through the herald's beacon and leave normal corpses on death.
 	id = "clockwork golem servant"
+	species_traits = list(NOBLOOD,NO_UNDERWEAR,NOEYES,NOSTOMACH,NOLIVER,NOGENITALS,NOAROUSAL,NOTRANSSTING,ROBOTIC_LIMBS,HAS_BONE)
```

Comment:
kinda wacky that noscrap golems would have robotic limbs, but the actual ones would not

Reply:
I didn't touch the regular clockwork golems because I'd thought touching those would be better done in another PR, but I can go ahead and equalize the two. My main thought process for this was that servant golems are more similar to synthetics, and the regular ones are just golems, made from brass. I dunno.